templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} the car</h1>
+  <form action="" method="post" novalidate>

```

Comment:
Consider adding an `action` attribute to the form tag to explicitly specify the URL for the form submission. This can help prevent issues if the template is reused or modified.